openedx/core/lib/xblock_serializer/block_serializer.py

```diff
@@ -21,20 +21,23 @@ class XBlockSerializer:
     """
     static_files: list[StaticFile]
     tags: TagValuesByObjectIdDict
+    olx_node: etree.Element
+    olx_str: str
 
-    def __init__(self, block, write_url_name=True, fetch_asset_data=False):
+    def __init__(self, block, write_url_name=True, fetch_asset_data=False, add_upstream=False):
         """
         Serialize an XBlock to an OLX string + supporting files, and store the
         resulting data in this object.
         """
         self.write_url_name = write_url_name
+        self.add_upstream = add_upstream
 
         self.orig_block_key = block.scope_ids.usage_id
         self.static_files = []
         self.tags = {}
-        olx_node = self._serialize_block(block)
+        self.olx_node = self._serialize_block(block)
 
-        self.olx_str = etree.tostring(olx_node, encoding="unicode", pretty_print=True)
+        self.olx_str = etree.tostring(self.olx_node, encoding="unicode", pretty_print=True)
```

Comment:
Are these changes required now?

Reply:
I removed the `add_upstream` here. The reference to the `olx_node` is used on the `ContainerSerializer`